api/design/catalog.go

```diff
@@ -26,18 +26,21 @@ var _ = Service("catalog", func() {
 	Error("not-found", ErrorResult, "Resource Not Found Error")
 
 	Method("Refresh", func() {
-		Description("Refreshes Tekton Catalog")
+		Description("Refresh a Tekton Catalog by its name")
```

Comment:
I think since we are having this for multicatalog support it should be as 
`Refresh a Catalog by it's name`